Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -1972,7 +1972,7 @@ namespace OpenInfraPlatform {
 
 				/**********************************************************************************************/
 				/*! \brief Calculates a trimming point on the curve.
-				* \tparam curve						A pointer to data from one of curves.
+				* \tparam TCurve						A pointer to data from one of curves.

```

Comment:
`TCurve` is not a pointer, rather it is a curve type. `curve` on the other hand, is the pointer to an instance of that curve type

Reply:
Resolved: b02d4d7